### DISCUSSION TITLE
Error after trying to install inventree_kicad plugin on docker

### DISCUSSION THREAD

**Discussion by martin323232** [inventree/InvenTree#7460] [Q&A]
Hello,

I tried to install the inventree_kicad plugin on inventree 11.3 with brother plugin 0.6.0. 

Following install guide on plugin page, after installing I restarted the server. 
The Brother lablel plugin did not load correctly with the error "missing pil.imagechops". Managed to sort this error out with downgrading Pillow to version 9.5 in docker. 
Can select labelprinter, webpage says its working but nothing is printed. 

After restart of entire server now docker keeps restarting the container, logs show this error.
`    import part.models
  File "/home/inventree/InvenTree/part/models.py", line 33, in <module>
    from stdimage.models import StdImageField
  File "/root/.local/lib/python3.9/site-packages/stdimage/__init__.py", line 1, in <module>
    from .models import JPEGField, StdImageField  # NOQA
  File "/root/.local/lib/python3.9/site-packages/stdimage/models.py", line 148, in <module>
    class StdImageField(ImageField):
  File "/root/.local/lib/python3.9/site-packages/stdimage/models.py", line 170, in StdImageField
    "resample": Image.ANTIALIAS,
AttributeError: module 'PIL.Image' has no attribute 'ANTIALIAS'`

There has been no reason to update server since everything have worked perfectly. 
Would like to get label printer to work again, any tips on how I can do this?
Have version backups of inventree-data folder if that helps.

Martin

**Comment by matmair** [comment]
We only support the current release cycle - 0.15.x

**Reply by martin323232** [reply]
ok
Can I update directly to 0.15 or do I need to go trough multiple versions?

**Reply by matmair** [reply]
Yes, you should be able to just update
